aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -116,6 +116,26 @@ If opting to remove a module with a setting in *web.config*, unlock the module a
     </system.webServer>
    </configuration>
    ```
+   
+In order to manipulate modules for IIS Express using *web.config*, you must modify *applicationHost.config* to unlock the `<modules>` section:
+
+1. Navigate to *{APPLICATION ROOT}\\.vs\config* and open the *applicationhost.config* file.
+
+1. Locate the `<section>` element for IIS modules and change `overrideModeDefault` from `Deny` to `Allow`:
+
+   ```xml
+   <section name="modules" 
+            allowDefinition="MachineToApplication" 
+            overrideModeDefault="Allow" />
+   ```
+   
+1. Locate the `<location path="" overrideMode="Allow"><system.webServer><modules>` section. For any modules that you wish to remove, set `lockItem` from `true` to `false`. In the following example, the CGI Module is unlocked:
+
+   ```xml
+   <add name="CgiModule" lockItem="false" />
+   ```
+   
+1. After the `<modules>` section and individual modules are unlocked, you're free to manipulate the modules in the app's local *web.config* file for IIS Express invoked by Visual Studio.
```

Comment:
Once the modules are unlocked, the modules in the app's local *web.config* file can be manipulated.

What does it mean to manipulate a file?

>  for IIS Express invoked by Visual Studio.

What does that mean?

web.config is read by IIS Express. IIS Express is launched from VS?
Why would you want to manipulate the files?